styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -371,13 +378,15 @@ public void create(Environment environment) {
 
     final Cleaner cleaner = new Cleaner(dockerRunner);
 
-    restoreState(storage, outputHandlers, stateManager, dockerRunner);
+    // TODO: is this still needed?
+    dockerRunner.restore();
```

Comment:
This should still be needed for the reasons explained here https://github.com/spotify/styx/pull/380/files#diff-a3fabd4f5e1e00cf48f40dfdfa3356f1L463

Reply:
Should restore that comment.